specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -221,6 +221,9 @@ model AcsSmsReceivedEventData extends AcsSmsEventBaseProperties {
   /** The time at which the SMS was received */
   @madeRequired(ServiceApiVersions.v2024_01_01)
   receivedTimestamp: utcDateTime;
+
+  /** Number of segments in the message */
+  segmentCount?: int32;
```

Comment:
When would this be excluded from the payload? Does that mean there are no segments?

Reply:
Hi Josh,
Events can receive payload with no segment property. We do add a default value to the payload but wanted to keep it optional throughout.

Reply:
Please let me know if you have any questions. Thanks Josh.

Reply:
> We do add a default value to the payload but wanted to keep it optional throughout.

I don't really understand then. If a default value is added, then how is it optional?

Reply:
Let me confirm with the SMS team to make sure. Sorry for the confusion Josh.

Reply:
Hello Josh,
Goe reply from SMS team. The SegmentCount is a required property and int32 datatype. Will make changes and upload the repos.

Reply:
Hello Josh, 
The changes are implemented. Please review and let me know if you have any questions. Thank you Josh.